### DISCUSSION TITLE
Moes MS-105Z doesnt bind to zigbee wireless switch in Z2M

### DISCUSSION THREAD

**Discussion by arveeproject** [Koenkk/zigbee2mqtt#15803] [Devices]
Hi,

I am Roland and new here. I started to work with Z2M in the last couple of months and am still on the basics of using Z2M.

I managed to bind Zigbee light bulbs to Zigbee wireless switches and also the Moes MS-Z104Z Zigbee Switch directly to Zigbee wireless switches without any problem.

Now i am starting to use Moes MS-105Z Zigbee Dimmers Switches, but i am not able to bind them to a Zigbee wireless switch. When i am trying to bind it, i get the instant Message "Noting to Bind". Why isn't this possible? 

I also see when i connect the Moes MS-105Z Dimmer Switch to Z2M, Z2M recognizes the dimmer switch after the interview as a TuYa TS0601_dimmer_1 and has the picture and description of that TuYa Switch. When i connect an Moest MS-104Z after the interview it has the exact picture and description in Z2M. Is it maybe possible that it is wrongfully put in Z2M as a device?

Awaiting some helpful answers.

With kind regards,

Roland

**Comment by Steini1984** [comment]
I opened a issue for this. I have been trying to mess with this my self, but have not gotten any further yet:
https://github.com/Koenkk/zigbee2mqtt/issues/15804

**Reply by arveeproject** [reply]
Good to hear and Thanks. The issue you opend explains exactly what happens in my case.

**Reply by boojew** [reply]
Seems this is still an issue :(
